tests/api/test_import_export.py

```diff
@@ -0,0 +1,136 @@
+from io import BytesIO
+from os import path
+
+from flask import url_for
+
+from app import alias_utils, s3
+from app.extensions import db
+from app.models import User, CustomDomain, Mailbox, Alias, AliasMailbox, ApiKey, File, BatchImport
+from app.import_utils import handle_batch_import
+from app.utils import random_string
+
+def test_export(flask_client):
+    # Create users
+    user1 = User.create(
```

Comment:
There's this method that will create a user and logs this user in so we don't have to create an api key in tests:

https://github.com/simple-login/app/blob/38790fdc84c26b9a09615ee7808dac982ab39361/tests/utils.py#L8

For example it's used here 
https://github.com/simple-login/app/blob/2b4dc3cdccea4f27898bb86e8f541efef15b2978/tests/dashboard/test_custom_alias.py#L33